hw/top_earlgrey/data/ip/chip_flash_ctrl_testplan.hjson

```diff
@@ -0,0 +1,270 @@
+              accessibility of the corresponding partition depending on the signal value.
+            '''
+      features: ["FLASH_CTRL.INFO.OWNER_PARTITION"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_flash_ctrl_lc_rw_en"]
+    }
+    {
+      name: chip_sw_flash_lc_iso_part_sw_rd_en
+      desc: '''Verify the lc_iso_part_sw_rd_en signal from LC ctrl.
+
+            - Transition from DEV to PROD to ESCALATION/SCRAP state via OTP and verify
+              that this LC signal transitions from 0 to 1 and back to 0. Verify that the SW
+              accessibility of the corresponding partition depending on the signal value.
+            '''
+      features: ["FLASH_CTRL.INFO.ISOLATED_PARTITION"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_flash_ctrl_lc_rw_en"]
+    }
+    {
+      name: chip_sw_flash_lc_iso_part_sw_wr_en
+      desc: '''Verify the lc_creator_seed_sw_wr_en signal from LC ctrl.
+
+            - Transition from TEST_LOCKED to DEV/PROD to ESCALATION/SCRAP state via OTP and verify
+              that this LC signal transitions from 0 to 1 and back to 0. Verify that the SW
+              accessibility of the corresponding partition depending on the signal value.
+            '''
+      features: ["FLASH_CTRL.INFO.ISOLATED_PARTITION"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_flash_ctrl_lc_rw_en"]
+    }
+    {
+      name: chip_sw_flash_lc_seed_hw_rd_en
+      desc: '''Verify the lc_seed_hw_rd_en signal from LC ctrl.
+
+            - Transition from TEST_LOCKED to DEV/PROD to ESCALATION/SCRAP state via OTP and verify
+              that this LC signal transitions from 0 to 1 and back to 0. Verify that the flash ctrl
+              does (or does not) read the creator and owner partitions to fetch the seeds for the
+              keymgr.
+            '''
+      features: ["FLASH_CTRL.INIT.ROOT_SEEDS"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_flash_ctrl_lc_rw_en"]
+    }
+    {
+      name: chip_sw_flash_lc_escalate_en
+      desc: '''Verify the lc_escalate_en signal from LC ctrl.
+
+            - Trigger an LC escalation signal by generating an alert.
+            - Verify that all flash accesses are disabled when the escalation kicks in.
+            - Confirm flash accesses are disabled by erroing if the device executes the ISR.
+            - Use assertion based connectivity check to prove that this signal is connected to the
+              flash ctrl.
+
+            X-ref with chip_sw_all_escalation_resets.
+            '''
+      features: ["FLASH_CTRL.ESCALATION"]
+      stage: V2
+      si_stage: SV3
+      lc_states: []
+      tests: ["chip_sw_all_escalation_resets"]
```

Comment:
One thing I failed to notice last time: every SiVal test case should have a `bazel: []` field, like here: https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/data/ip/chip_kmac_testplan.hjson;drc=8ef4c5525d76b42009f75c6f153759e0766c4487;l=113

This will be similar to the `tests: []` field, except instead of being picked up by dvsim, it will be picked up by Bazel / maybe some future scripting tooling.

Reply:
Noted, I could put an place holder for the moment, but it might be better I update with real bazel target.
Clearly some tests may not qualified for the SiVal platform.